napari/types.py

```diff
@@ -129,6 +136,13 @@ def __getattr__(self, name: str) -> Any:
 LayerDataTuple = NewType("LayerDataTuple", tuple)
 
 
+# This namedtuple increases code readability, but is only for internal use
+class LayerDataTupleNamed(NamedTuple):
+    data: Union[ArrayLike, Sequence[ArrayLike]]
+    attributes: Optional[Dict] = None
+    layer_type: Optional[LayerTypeName] = None
```

Comment:
None is not valid here, if it's a length 3 tuple, it _has_ to be a string

Reply:
I've removed the optional, but not put in a default - for internal use I think it might be best to force this to be explicit, and not have a deafult of `image`?